src/controllers/bookings-controller.js

```diff
@@ -74,5 +74,42 @@ async function updateBookingAddress(req, res, next) {
         next(err);
     }
 }
+async function getBookingsByEmployeeId(req, res, next) {
+    const { eid } = req.params;
+    try {
+        const bookings = await db.Bookings
+            .find({ employeeId: eid })
+            .select("-__v -contactInfo")
+            .sort({ createdAt: -1 })
+            .lean()
+            .exec();
+        res.status(200).send({
+            data: bookings,
+            error: null
+        });
+    } catch (err) {
+        next(err);
+    }
+}
+async function setStatus(req, res, next) {
+    const { propertyId, status } = req.params;
+    if (status == "pending" || status == "rejected" || status == "accepted") {
```

Comment:
I would pass the status on the body of the request and build a Joi validation to validate the string so you don't need to write and maintain multiple if/else statements whenever you add a new status